src/chrome/content/rules/newtalk.tw.xml

```diff
@@ -0,0 +1,22 @@
+<!--
+	http://.+\.newtalk\.tw/	redirect to ^ by server, and most of them are Invalid certificate.
+-->
+
+<ruleset name="newtalk.tw">
+
+	<!--	Complications:	-->
+	<target host="*.newtalk.tw" />
+		<test url="http://abcdefg123456789.newtalk.tw/" />
+		<test url="http://ap1.newtalk.tw/" />
+		<test url="http://www.newtalk.tw/" />
+
+	<rule from="^http://\w+\.newtalk\.tw/" to="https://newtalk.tw/" />
```

Comment:
`\w+` doesn't match with dash, such as abcdefg-123456789.newtalk.tw. I'd suggest you to use `.+`.